packages/engine/Source/Scene/PostProcessStageLibrary.js

```diff
@@ -669,6 +670,7 @@ PostProcessStageLibrary.createAcesTonemappingStage = function (
     fragmentShader: fs,
     uniforms: {
       autoExposure: undefined,
+      exposure: 1,
```

Comment:
As per the [coding guide](https://github.com/CesiumGS/cesium/blob/main/Documentation/Contributors/CodingGuide/README.md#basic-code-construction)

```suggestion
      exposure: 1.0,
```

to make it clear this is a float. Also applies below.